staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go

```diff
@@ -352,11 +350,12 @@ func IsValidPortName(port string) []string {
 }
 
 // IsValidIP tests that the argument is a valid IP address.
-func IsValidIP(value string) []string {
+func IsValidIP(fldPath *field.Path, value string) field.ErrorList {
```

Comment:
huh... this function directly changing API validation but not being under API review is sort of scary

it looks like this is only used (outside of tests) in pkg/apis/core/validation. can we move this function there?

Reply:
fun times. we could also put the validation packages under API review with an OWNERS file like this:

```
# See the OWNERS docs at https://go.k8s.io/owners

# Disable inheritance as this is an api owners file
options:
  no_parent_owners: true
approvers:
  - api-approvers
reviewers:
  - api-reviewers
labels:
  - kind/api-change
```

that's probably easier than relocating stuff

Reply:
do you want me to do that in this PR?

Reply:
nah, opened https://github.com/kubernetes/kubernetes/pull/123164 to do that